media_driver/agnostic/gen11/codec/hal/codechal_kernel_header_g11.h

```diff
@@ -29,6 +29,7 @@
 
 #include "codechal_encoder_base.h"
 
+#ifndef _FULL_OPEN_SOURCE
```

Comment:
no. Please, move these ifdefs inside structs.

Why you drop part of the structures declarations?

Reply:
the reason is that we have different kernel structure, we only have DS and HME open sourced, the structure should align with the kernels enabled